src/core/ext/filters/client_channel/retry_filter.cc

```diff
@@ -1593,6 +1596,10 @@ void RetryFilter::CallData::CallAttempt::BatchData::
     call_attempt_->recv_trailing_metadata_error_ = error;
     return;
   }
+  // Copy transport stats to be delivered up to the surface.
```

Comment:
Note that this is a `move`, not a `copy`

Reply:
I'll fix this in a subsequent PR, to avoid delaying this one further.